libut/tests/Makefile

```diff
@@ -4,7 +4,8 @@ OBJS=$(patsubst %,%.o,$(PROGS))
 
 CFLAGS += -I../include
 CFLAGS += -g
-CFLAGS += -Wall -Wextra
+CFLAGS += -Wall -Wextra -Werror -fPIC
+CFLAGS += $(OPTFLAGS)
```

Comment:
`OPTFLAGS` is an environment variable $RPM_OPT_FLAGS visible when building a RPM. that is, packager can pass it  `make OPTFLAGS="$RPM_OPT_FLAGS"` in specfile (like debian rules). On non-RPM system, it will be nothing. It will not harm. And packagers will create such downstream patches anyway if we don't add it upstream.